src/lib/helpers/shortenAddress.ts

```diff
@@ -0,0 +1,20 @@
+import { getAddress } from "@ethersproject/address"
+
+// returns the checksummed address if the address is valid, otherwise returns false
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+export function isAddress(value: any): string | false {
```

Comment:
Just use `ethers.utils.isAddress`?

Reply:
isAddress behaves differently of projectethers isAddress lib.